cloudify/workflows/workflow_context.py

```
@@ -1031,16 +1053,19 @@ def stop(self):
     def add_task(self, task):
         self._local_tasks_queue.put(task)
 
-    def _process_local_task(self):
+    def _process_local_task(self, workflow_ctx):
         # see CFY-1442
```

Actually i think try/finally isn't needed here at all, because the `task()` call is inside a `try/except: pass` block anyway... how about this new version?